tests/interactive/test_app.py

```diff
@@ -42,7 +42,7 @@ async def test_inputs():
         assert app.output_text_area.text == "No input"
         assert app.current_module is None
 
-        # Test inccorect input
```

Comment:
Just occurred to me this might have been a joke that I wasn't funny enough to understand